samples/crypto/psa_tls/README.rst

```diff
@@ -29,6 +29,202 @@ Overview
 The sample can act as either a network server or a network client.
 By default, the sample is configured to act as a server.
 
+TLS version support
+===================
+
+The sample supports both TLS v1.2 and TLS v1.3.
+See the following table for the support overview for each compatible board target.
+The table mirrors the test setup used for TLS verification.
+
+.. tabs::
+
+   .. tab:: TLS v1.2
+
+      .. list-table::
+         :header-rows: 1
+         :widths: auto
+
+         * - Board target
+           - Backend
+           - CMSE supported
+           - DTLS supported
+           - Limitations / not supported
+         * - ``nrf52840dk/nrf52840``
+             ``nrf9160dk/nrf9160``
+             ``nrf9151dk/nrf9151``
+           - :ref:`cc3xx_legacy<nrf_security_drivers_legacy>`
```

Comment:
@magnev , can you double check links in this table? I'm not sure if the legacy links are correct. Thanks.